lexicons/com/atproto/label/label.json

```diff
@@ -0,0 +1,23 @@
+{
+  "lexicon": 1,
+  "id": "app.bsky.label.label",
+  "defs": {
+    "main": {
+      "type": "record",
+      "description": "Metadata tag on an atproto resource (eg, repo or record)",
+      "key": "tid",
+      "record": {
+        "type": "object",
+        "required": ["src", "uri", "val", "ts"],
+        "properties": {
+          "src": {"type": "string", "maxLength": 128},
```

Comment:
Intuitively this seems like a reasonable number, but if there's a `maxLength` to bump up higher it might be this one.  For did:plc this length is good, but for other did methods (in particular did:web), it's possible we'd want to give some more leeway.

Reply:
I bumped `src` length from 128 to 1024, and CID from 128 to 256 for wiggle room.
I'm sort of disposed to just not add these length restrictions here at all, unless we decide to also add in other places (like `com.atproto.repo.strongRef`). Maybe as part of a "hardening" last pass over these lexicons.